app/controllers/v1/messages_controller.rb

```diff
@@ -0,0 +1,6 @@
+class V1::MessagesController < ApplicationController
+  def index
+    @messages = Message.all
+    render json: { message: @messages.sample.message }
+  end
```

Comment:
- [ ] Good job so far 👍 Kindly create a database table for storing different greetings. This is done by creating the migration file for the table and it's corresponding model 💯 In that way, the random message will be gotten from your database 👍